File: Backend/fl-project-service/fl-project-service/src/main/java/com/fl/project/model/response/BidResponse.java
package com.fl.project.model.response;

import com.fasterxml.jackson.annotation.JsonIgnore;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class BidResponse {
    private int bidId;
    private int projectId;
    private int freelancerId;
    private int amount;
    private String status;
    private String description;
    private String createdDate;
}

File: frontend/fl/src/app/shared/model/bidResponse.ts
import { userResponse } from './userResponse';

export interface bidResponse {
  bidId: number;
  projectId: number;
  freelancerId: number;
  amount: number;
  deliveryDays: number;
  description: string;
  status: string;
  createdDate: string;
  freelacerDetails: userResponse;
}
